### ISSUE TITLE
[a11y] 9.2.4.7 Current position of the focus clearly

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10735]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-2-4-7-aktuelle-position-des-fokus-deutlich

### Personal 

- [ ] The keyboard focus on dark blue buttons is very difficult to recognize.
This problem also affects slider elements and toggle buttons.
![image](https://github.com/owncloud/web/assets/132444655/99decbb2-3b7a-4289-97ee-8ba7d8f25a6f)

- [ ] With some drop-down menus and "select" elements, the list elements are
are very difficult to recognize as focused. As the focus is only highlighted by
a color change, the contrast ratio between the focused and unfocused state should be at least
unfocused state should be at least 3:1.
![image](https://github.com/owncloud/web/assets/132444655/21e71a37-94f2-445c-a2b7-e7a8b6ed7ac5)

### Shares

- [ ] The buttons in the table columns "Shared by" and "Shared with" have no visible keyboard focus.
have no visible keyboard focus.
![image](https://github.com/owncloud/web/assets/132444655/76b15ad6-c875-410b-9333-d7fbbe0bd4e0)

- [ ] See comments on Personal

### Spaces

- [ ] The checkboxes on dark backgrounds have a keyboard focus that is difficult to recognize.
keyboard focus. The "Deactivated" button also has an unrecognizable keyboard focus.
keyboard focus.
![image](https://github.com/owncloud/web/assets/132444655/2ba0d6a8-b209-49e8-950b-3c349286214b)

- [ ] See comments on Personal

### Account

- [ ] The "Receive notification mails" checkbox does not have a visible keyboard focus.

**Comment by pascalwengerter** [comment]
@tbsbdr needs a concept/decision regarding "focus color" and "background color for currently selected element in OcSelect"

**Comment by tbsbdr** [comment]
I'd suggest the following from:
https://www.w3.org/WAI/WCAG21/Techniques/css/C40#example-1-a-thick-two-color-indicator


```
*:focus {
	/* inner indicator */
	outline: 2px #F9F9F9 solid;
	outline-offset: 0;
	/* outer indicator */
	box-shadow: 0 0 0 4px #193146;
}
```
